docs/gettingstarted/modfiles.md

```diff
@@ -167,6 +171,7 @@ There must be a 1-to-1 matching of mods in the `mods.toml` file and `@Mod` entry
 [events]: ../concepts/events.md
 [features]: #features
 [group]: #the-group-id
+[i18n]: ../concepts/internationalization.md
```

Comment:
Link to the specific section in internationalization.

Reply:
:+1: